Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGallery.swift

```diff
@@ -117,10 +105,11 @@ public struct BasemapGallery: View {
                 } message: { item in
                     Text(item.message)
                 }
-                .frame(height: min(contentHeight, geometry.size.height))
-                .esriBorder()
+                .frame(
+                    width: geometry.size.width,
+                    height: geometry.size.height
```

Comment:
Use the full height available so content isn't trimmed while scrolling.

Before:
![2023-03-30 12 50 03](https://user-images.githubusercontent.com/16397058/228954132-afe20183-3f5b-4e39-a062-9c84bd2aa741.gif)

After:
![2023-03-30 12 49 21](https://user-images.githubusercontent.com/16397058/228954176-42a1f6b9-4726-430b-84d9-fb5e9eea0a8e.gif)